earthpy/tests/test_plot_rgb.py

```diff
@@ -63,6 +63,7 @@ def test_1band(rgb_image):
                            order with bands first""",
     ):
         plot_rgb(a_rgb_image[1])
+    plt.close()
```

Comment:
See how weird this is! When you squash a pr it seems like sometimes when you add those commits to another PR they show up as . "changes" in the PR even tho they are in master. such a weird GH thing.

Reply:
For sure - let's make sure we remember to squash merge this PR!